### DISCUSSION TITLE
Any recommendations on how to change `_layouts/about` so that the context text does not wrap around the image?

### DISCUSSION THREAD

**Discussion by chnsh** [alshedivat/al-folio#1334] [Q&A] [answered]
Hey everyone, 
I had a quick question - at the moment if the `about.md` content is longer than the image, it wraps around the image and renders below the image as well. I was wondering what would be a good way to format the layout such that the content is in its own column and does not wrap below the image?

Thanks

**Comment by george-gca** [comment]
One possible solution would be changing the layout in the [_layouts/about.html](https://github.com/alshedivat/al-folio/blob/c824d128673c000ef9f0e0e77a0ae30d2c4b4568/_layouts/about.html#L17). For example, for [my site](https://george-gca.github.io/) I [changed its location](https://github.com/george-gca/george-gca.github.io/blob/029c40ba8b917a74a98ee3ad878154ae096bb71b/_layouts/about.html#L10) in the code so it would occupy more space above.

**Comment by abhilesh** [comment, marked as answer]
I was facing a similar issue and managed to avoid the text wrapping underneath the profile picture on [my website](https://abhilesh.github.io/) by modifying my _layouts/about.html. You can see the changes made in [this commit on my repository.](https://github.com/abhilesh/abhilesh.github.io/commit/113cf6f4d161107600a9b62f10cb939db9ab3f78#diff-9a9434af40c4503e3ca4fbd0e87e85a1518b3d46f8e4b1ff44c6b43741aa5ca6)

Tested it on the browser and mobile layouts and no issues so far. Hope this helps!
